src/client/helpers/utils.tsx

```diff
@@ -171,6 +171,34 @@ export function dateObjectToISOString(value: DateObject) {
 	return date;
 }
 
+/**
+ * Remove trailing parantheses from a url string.
+ * @function removeTrailingParantheses
+ * @param {string} url - URL string.
+ * @returns {string} cleanUrl - sans any trailing parantheses.
+ */
+function removeTrailingParantheses(url: string, count: number): string {
+    return url.slice(0, -1*count);
+}
+
+/**
+ * Count trailing parantheses from a url string.
+ * @function countTrailingParantheses
+ * @param {string} url - URL string.
+ * @returns {number} count - count of trailing parantheses.
+ */
+function countTrailingParantheses(url: string): number {
+    let cnt: number = 0;
+    for (const c of url) {
+        if (c === '(') {
+            cnt += 1;
+        } else if (c === ')') {
+            cnt -= 1;
+        }
+    }
+	return Math.abs(cnt);
```

Comment:
I feel like this _probably_ shouldn’t be an absolute number in case it somehow does end up with a negative number? I suspect you’d want to get `-2` returned if it somehow counted two more `)` than `(` instead of `2`, which would suggest two more `(` than `)`?

I don’t know how `String.repeat()` would handle that, but… :shrug:

Reply:
E.g., how would this handle `https://da.wikipedia.org/wiki/Christian_Christensen_(forfatter)))))))))`? (I didn’t try and decipher the revised regex to see if this is ever something that could happen. Would be good with a test either way to ensure it works as expected/desired.)